test/integration/backward_compatible/parallel/map/NearCachedMapTest.js

```diff
@@ -102,7 +102,7 @@ describe('NearCachedMapTest', function () {
                 }
                 await map1.get('key1');
                 await map2.remove('key1');
-                const val = await TestUtil.promiseLater(1000, map1.get.bind(map1, 'key1'));
+                const val = await TestUtil.promiseLater(5000, map1.get.bind(map1, 'key1'));
```

Comment:
Didn't we apply some other changes to this part on other PRs?

Reply:
I am not sure this is left like this, let me rebase